vue-app/src/api/contributions.ts

```diff
@@ -69,40 +69,46 @@ export async function getContributionAmount(
   fundingRoundAddress: string,
   contributorAddress: string
 ): Promise<BigNumber> {
-  const fundingRound = new Contract(fundingRoundAddress, FundingRound, provider)
-  const filter = fundingRound.filters.Contribution(contributorAddress)
-  const events = await fundingRound.queryFilter(filter, 0)
-  const event = events[0]
-  if (!event || !event.args) {
+  const data = await sdk.GetContributionsAmount({
+    fundingRoundAddress: fundingRoundAddress.toLowerCase(),
+    contributorAddress,
+  })
+
+  if (!data.fundingRound?.contributors?.[0]?.contributions?.length) {
     return BigNumber.from(0)
   }
-  return event.args._amount
-}
 
-export async function isContributionWithdrawn(
```

Comment:
How are we able to remove this function? Does `getContributionAmount` now handle this?

Reply:
Ah, I think I see - `handleContributionWithdrawn` is the subgraph mappings handles this?

Reply:
Will add a flag to the subgraph schema for this so it is easier to query.